lib/gtest_1.8.1/src/gtest_main.cc

```diff
@@ -28,22 +28,10 @@
 // OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
 
 #include <stdio.h>
-
 #include "gtest/gtest.h"
 
-#include <stan/math/prim/arr/functor/mpi_cluster.hpp>
-
-
 GTEST_API_ int main(int argc, char **argv) {
-#ifdef STAN_MPI
-  // for MPI testing we test with all workers in listen mode. No
-  // output is generated from the workers.
-  stan::math::mpi_cluster cluster;
-  cluster.listen();
-  if (cluster.rank_ != 0)
-    return 0;
-#endif
-  printf("Running main() from gtest_main.cc\n");
+  printf("Running main() from %s\n", __FILE__);
```

Comment:
@wds15, do we still need the old code here? If I understand it correctly, this is necessary to run the MPI tests, right?

Reply:
Uh... maybe that's outdated and we can drop this; I need to check.

Reply:
I just checked and we do need this extra bit for the MPI tests to work.